automation/proxy_container/version.sh

```diff
@@ -7,7 +7,12 @@ function docker_tag () {
 
     HASH_FILES=(${BASH_SOURCE[0]} ${dockerfile} $current_dir/requirements.txt $current_dir/docker_user.pem $current_dir/docker_user.pem.pub)
     HASH_FILES+=($(find $current_dir/etc/ -type f))
-    local files_sum="$(md5sum -- ${HASH_FILES[@]} | awk {'print $1'})"
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        #if Running from Mac OSX the md5sum command output is a bit different
+      local files_sum="$(md5sum -- ${HASH_FILES[@]} | awk {'print $4'})"
```

Comment:
NIT: you can check this one 
https://raamdev.com/2010/mac-os-x-replicating-md5sum-output-format/#:~:text=The%20md5sum%20program%20is%20used,verify%20128%2Dbit%20MD5%20hashes.&text=Mac%20OS%20X%20is%20a,than%20the%20standard%20md5sum%20program.